Make scenes page list view checkbox fill entire cell

7dJx1qP opened the pull request:
The current checkbox inputs on the scenes page listview are hard to click, so I wrapped them in a label that fills the table cell. Clicking any part of the table cell will now check the input.

The blue outlines show the clickable area that will check the inputs:

Before
![image](https://user-images.githubusercontent.com/38586902/193654856-2eb0d5fd-aa9a-46a1-b2a3-ac9e60950aaa.png)

After
![image](https://user-images.githubusercontent.com/38586902/193655023-cde4a814-965b-43cf-aaf3-8c522bf77d1e.png)

WithoutPants commented:
Extending the checkbox to the edge of the cell looks pretty ugly to me.

My suggestion is to follow what's in this stackoverflow answer: https://stackoverflow.com/a/55157487

I'd also prefer doing this in css rather than using the bootstrap classes.

7dJx1qP commented:
Ok, I changed the bootstrap classes to css and added padding to the label.
![image](https://user-images.githubusercontent.com/38586902/194194543-23de61cd-c8cd-449c-ad12-3c906332dc15.png)
